opentelemetry-stdout/src/trace/exporter.rs

```diff
@@ -22,106 +20,136 @@ impl fmt::Debug for SpanExporter {
     }
 }
 
-impl SpanExporter {
-    /// Create a builder to configure this exporter.
-    pub fn builder() -> SpanExporterBuilder {
-        SpanExporterBuilder::default()
-    }
-}
-
 impl Default for SpanExporter {
     fn default() -> Self {
-        SpanExporterBuilder::default().build()
+        SpanExporter {
+            resource: Resource::default(),
+            is_shutdown: atomic::AtomicBool::new(false),
+            resource_emitted: false,
+        }
     }
 }
 
 impl opentelemetry_sdk::export::trace::SpanExporter for SpanExporter {
+    /// Write Spans to stdout
     fn export(&mut self, batch: Vec<export::trace::SpanData>) -> BoxFuture<'static, ExportResult> {
-        let res = if let Some(writer) = &mut self.writer {
-            (self.encoder)(writer, crate::trace::SpanData::new(batch, &self.resource)).and_then(
-                |_| {
-                    writer
-                        .write_all(b"\n")
-                        .map_err(|err| TraceError::Other(Box::new(err)))
-                },
-            )
+        if self.is_shutdown.load(atomic::Ordering::SeqCst) {
+            Box::pin(futures_util::future::ready(Err(TraceError::from(
+                "exporter is shut down",
+            ))))
         } else {
-            Err("exporter is shut down".into())
-        };
+            if self.resource_emitted {
+                print_spans(batch);
+            } else {
+                self.resource_emitted = true;
+                println!("Resource");
+                if self.resource.schema_url().is_some() {
+                    println!("\t Resource SchemaUrl: {:?}", self.resource.schema_url());
+                }
 
-        Box::pin(std::future::ready(res))
+                self.resource.iter().for_each(|(k, v)| {
+                    println!("\t ->  {}={:?}", k, v);
+                });
+
+                print_spans(batch);
+            }
+
+            Box::pin(futures_util::future::ready(Ok(())))
```

Comment:
Is the change from `std::future::ready()` to `futures_util::future::ready()` intentional? Does this work?
```suggestion
            Box::pin(std::future::ready(Ok(())))
```